Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -135,6 +166,148 @@ namespace AzToolsFramework
+            return m_widget;
+        }
+
+        void SetValueFromDom(const AZ::Dom::Value& node)
+        {
+            using AZ::DocumentPropertyEditor::Nodes::PropertyEditor;
+
+            auto propertyEditorSystem = AZ::Interface<AZ::DocumentPropertyEditor::PropertyEditorSystemInterface>::Get();
+            AZ_Assert(
+                propertyEditorSystem != nullptr,
+                "RpePropertyHandlerWrapper::SetValueFromDom called with an uninitialized PropertyEditorSystemInterface");
+            if (propertyEditorSystem == nullptr)
+            {
+                return;
+            }
+
+            m_proxyClassElement.m_attributes.clear();
+            for (auto attributeIt = node.MemberBegin(); attributeIt != node.MemberEnd(); ++attributeIt)
+            {
+                const AZ::Name& name = attributeIt->first;
+                if (name == PropertyEditor::Type.GetName() || name == PropertyEditor::Value.GetName() ||
+                    name == PropertyEditor::ValueType.GetName())
+                {
+                    continue;
+                }
+                AZ::Crc32 attributeId = AZ::Crc32(name.GetStringView());
+
+                const AZ::DocumentPropertyEditor::AttributeDefinitionInterface* attribute =
+                    propertyEditorSystem->FindNodeAttribute(name, propertyEditorSystem->FindNode(AZ::Name(GetHandlerName(m_rpeHandler))));
+                AZStd::shared_ptr<AZ::Attribute> marshalledAttribute;
+                if (attribute != nullptr)
+                {
+                    marshalledAttribute = attribute->DomValueToLegacyAttribute(attributeIt->second);
+                }
+                else
+                {
+                    marshalledAttribute = AZ::Reflection::WriteDomValueToGenericAttribute(attributeIt->second);
+                }
+
+                if (marshalledAttribute)
+                {
+                    m_proxyClassElement.m_attributes.emplace_back(attributeId, AZStd::move(marshalledAttribute));
+                }
+            }
+
+            AZ::Dom::Value value = AZ::DocumentPropertyEditor::Nodes::PropertyEditor::Value.ExtractFromDomNode(node).value();
+            m_proxyValue = AZ::Dom::Utils::ValueToType<WrappedType>(value).value();
+            m_rpeHandler.ReadValuesIntoGUI_Internal(GetWidget(), &m_proxyNode);
+            m_rpeHandler.ConsumeAttributes_Internal(GetWidget(), &m_proxyNode);
+
+            m_domNode = node;
+        }
+
+        QWidget* GetFirstInTabOrder() override
+        {
+            return m_rpeHandler.GetFirstInTabOrder_Internal(GetWidget());
+        }
+
+        QWidget* GetLastInTabOrder() override
+        {
+            return m_rpeHandler.GetLastInTabOrder_Internal(GetWidget());
+        }
+
+        static bool ShouldHandleNode(PropertyHandlerBase& rpeHandler, const AZ::Dom::Value& node)
+        {
+            using namespace AZ::DocumentPropertyEditor::Nodes;
```

Comment:
update: Better not to use `using namespace` here and just use `using <TYPE>` to bring into scope the type you need